pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[tool.poetry]
```

Comment:
Is this file being read by the github workflow? I'm not familiar with toml/poetry so bear with me :)

Reply:
Oops, I use poetry on my end and accidentally committed those files. Will remove!